src/odemis/driver/technolution.py

```diff
@@ -89,7 +88,7 @@ def convert2Bits(value, value_range):
     :param value_range (tuple): min, max values of the range that is provided for the input value.
     :return(tuple/array): Converted to INT16 range with same shape as the input value.
```

Comment:
When using this function for creation of the setpoints the float 'feature' is used. Because you substract 'bits' values or use them as input for linspace it is better/more precise to round them only at the end just before sending them to the ASM.